io.catenax.order_status/1.0.0/OrderStatus.ttl

```diff
@@ -0,0 +1,100 @@
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.order_status:1.0.0#>.
+@prefix ext-planned: <urn:bamm:io.catenax.part_as_planned:1.0.0#>.
+@prefix ext-relationship: <urn:bamm:io.catenax.assembly_part_relationship:1.1.1#>.
+@prefix ext-certificate: <urn:bamm:io.catenax.esr_certificates.esr_certificate:1.0.1#>.
+
+:OrderStatus a bamm:Aspect;
+    bamm:properties (:manufacturerOrderId [
+  bamm:property :customerOrderId;
+  bamm:optional "true"^^xsd:boolean
+] :orderPositionStatuses);
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "Status of Order"@en;
+    bamm:description "This aspect represents the latest status of an order, that describes the business relationship between a customer and a manufacturer along the value chain. "@en.
+:manufacturerOrderId a bamm:Property;
```

Comment:
There is an ongoing discussion on defining shared aspects for several catenax specific ids. Maybe manufacturerOrderId is one and could be used here.